src/ralph/lib/mixins/fields.py

```diff
@@ -169,3 +171,38 @@ def get_limit_models(self):
         Returns Model class list from limit_models.
         """
         return [get_model(model) for model in self.limit_models]
+
+
+class TagWidget(forms.TextInput):
+    def render(self, name, value, attrs=None):
+        if value is not None and not isinstance(value, six.string_types):
+            value = ', '.join(sorted([
+                (t if ',' not in t else '"%s"' % t) for t in value
```

Comment:
type of `value` in this method could be either `None`, `str` or `list (iterable)`